docs/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "@primer/gatsby-theme-doctocat": "*",
-    "@primer/react": "^34.5.0",
+    "@primer/react": "^35.3.0-rc.14a76ea9",
```

Comment:
Waiting on the next Primer React release before merging this PR

https://github.com/primer/react/pull/2056